.evergreen/install-dependencies.sh

```diff
@@ -16,22 +20,91 @@ mkdir -p ${NODE_ARTIFACTS_PATH}
 mkdir -p ${NPM_CACHE_DIR}
 mkdir -p "${NPM_TMP_DIR}"
 
-# install Node.js
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.35.3/install.sh | bash
-[ -s "${NVM_DIR}/nvm.sh" ] && \. "${NVM_DIR}/nvm.sh"
-nvm install --no-progress --lts=${NODE_LTS_NAME}
+case $NODE_LTS_NAME in
+  "argon")
+    VERSION=4
+    ;;
+  "boron")
+    VERSION=6
+    ;;
+  "carbon")
+    VERSION=8
+    ;;
+  "dubnium")
+    VERSION=10
+    ;;
+  "erbium")
+    VERSION=12
+    ;;
+  "fermium")
+    VERSION=14
+    ;;
```

Comment:
Does this add fermium to our matrix or just gets us ready to use it?

Reply:
This just gets us ready to use it in the future. We basically have to duplicate the `--lts` feature of `nvm` because `nvm-windows` doesn't have it.